pages/banner-iframe.js

```diff
@@ -213,46 +216,50 @@ class BannerIframe extends React.Component {
       }
     }
 
-    return { collectiveSlug, id, style, useNewFormat: Boolean(useNewFormat) };
+    return { collectiveSlug, id, style, useNewFormat: parseToBoolean(useNewFormat) };
   }
 
   static propTypes = {
     collectiveSlug: PropTypes.string, // from getInitialProps, for addCollectiveBannerIframeData
     id: PropTypes.string, // from getInitialProps
-    style: PropTypes.object, // from getInitialProps
+    style: PropTypes.string, // from getInitialProps
     data: PropTypes.object.isRequired, // from withData
     useNewFormat: PropTypes.bool,
   };
 
-  constructor(props) {
-    super(props);
-    this.onChange = this.onChange.bind(this);
-  }
-
   componentDidMount() {
-    this.onChange();
+    this.onSizeUpdate();
+    window.addEventListener('resize', this.onSizeUpdate);
+    this.updateSizeInterval = setInterval(this.onSizeUpdate, 1000);
```

Comment:
This is a dirty hack, but `requestAnimationFrame` was insufficient to ensure we got the proper size; `getBoundingClientRect` was always returning the previous one. We should ideally figure a better solution, but I don't have the time to investigate deeper right now.